sdk/storage/azure-storage-file-datalake/tests/test_file.py

```diff
@@ -907,6 +911,42 @@ def test_rename_file_with_non_used_name(self, datalake_storage_account_name, dat
         self.assertEqual(data, data_bytes)
         self.assertEqual(new_client.path_name, "newname")
 
+    @DataLakePreparer()
+    def test_file_encryption_scope(self, datalake_storage_account_name, datalake_storage_account_key):
+        # Arrange
+        url = self.account_url(datalake_storage_account_name, 'dfs')
+        self.dsc = DataLakeServiceClient(url, credential=datalake_storage_account_key, logging_enable=True)
+        self.config = self.dsc._config
+        self.file_system_name = self.get_resource_name('filesystem')
+        file_name = 'testfile'
+        file_system = self.dsc.get_file_system_client(self.file_system_name)
+        try:
+            file_system.create_file_system(file_system_encryption_scope=TEST_FILE_SYSTEM_ENCRYPTION_KEY_SCOPE)
+        except ResourceExistsError:
+            pass
+
+        file_client = file_system.create_file(file_name)
+        props = file_client.get_file_properties()
+
+        # Assert
+        self.assertTrue(props)
+        self.assertIsNotNone(props['encryption_scope'])
+        self.assertEqual(props['encryption_scope'], TEST_FILE_SYSTEM_ENCRYPTION_KEY_SCOPE.default_encryption_scope)
+
+    @pytest.mark.live_test_only
+    @DataLakePreparer()
+    def test_hns_encryption_scope_sas(self, datalake_storage_account_name, datalake_storage_account_key):
+        self._setUp(datalake_storage_account_name, datalake_storage_account_key)
+        token = generate_account_sas(
+            self.dsc.account_name,
+            self.dsc.credential.account_key,
+            ResourceTypes(object=True),
+            AccountSasPermissions(write=True, read=True, create=True, delete=True),
+            datetime.utcnow() + timedelta(hours=5),
+            encryption_scope=TEST_HNS_SAS_ENCRYPTION_SCOPE,
+        )
+        self.assertTrue(token.__contains__(TEST_HNS_SAS_ENCRYPTION_SCOPE))
```

Comment:
`token` is just a string, right? If so, don't use `__contains__` and instead do `TEST_HNS_SAS_ENCRYPTION_SCOPE in token`.
Because Python is super intuitive like that, and you can literally type what you mean :D